autoPyTorch/pipeline/components/preprocessing/image_preprocessing/normalise/base_normalizer.py

```diff
@@ -4,9 +4,16 @@
 
 from autoPyTorch.pipeline.components.preprocessing.image_preprocessing.base_image_preprocessor import \
     autoPyTorchImagePreprocessingComponent
+from autoPyTorch.utils.common import FitRequirement
 
 
 class BaseNormalizer(autoPyTorchImagePreprocessingComponent):
+    _fit_requirements = [FitRequirement('channelwise_mean', np.ndarray),
```

Comment:
With this implemented, we can move parts of the `check_requirements` method to the base Component to prevent redundant code, so that the base component goes through `_fit_requirements` and checks if the key is available in X and checks the type. We can do it now or add an issue to remind ourselves

Reply:
okay I'll do that now.